### PR TITLE
Added ctx param to video_stop/video_stop_source and set ctx to null

### PR DESCRIPTION
**Pull request by juha-h**
when video source is stopped in order to solve issue https://github.com/baresip/baresip/issues/1169.

### CODE REVIEW

File: src/video.c [hunk 1]
```diff
@@ -1206,14 +1206,15 @@ int video_start_display(struct video *v, const char *peer)
...
-static void video_stop_source(struct video *v)
+static void video_stop_source(struct video *v, struct media_ctx **ctx)
 {
 	if (!v)
 		return;
 
 	debug("video: stopping video source ..\n");
 
 	v->vtx.vsrc = mem_deref(v->vtx.vsrc);
+	*ctx = NULL;
```
Comment (alfredh) [inline src/video.c:1217]: to make it more robust, please check if `ctx` is not null

**Comment by alfredh** [comment]
can you confirm that this fixes the crash on your Android app ?

**Comment by juha-h** [comment]
Alfred E. Heggestad writes:

> can you confirm that this fixes the crash on your Android app ?

Have tested tens of times to turn video on/off without crash.

**Comment by alfredh** [comment]
thanks !

**Pull request merged**